src/static.css

```diff
@@ -18,3 +20,8 @@
 :root[style*="--black"]:not([style*="--content-fg-secondary"]) { --content-fg-secondary: rgba(var(--black), 0.65); }
 :root[style*="--black"]:not([style*="--content-fg-tertiary"]) { --content-fg-tertiary: rgba(var(--black), 0.4); }
 :root[style*="--white"]:not([style*="--modal"]) { --modal: rgb(var(--white)); }
+:root[style*="--deprecated-accent"]:not([style*="--chrome-ui"]) { --chrome-ui: rgb(var(--deprecated-accent)); }
+:root[style*="--deprecated-accent"][style*="--white-on-dark"]:not([style*="--chrome-ui-hover"]) { --chrome-ui-hover: color-mix(in srgb, rgb(var(--deprecated-accent)) 100%, rgb(var(--white-on-dark)) 25%); }
```

Comment:
- I believe this is the same as `color-mix(in srgb, rgb(var(--deprecated-accent)), rgb(var(--white-on-dark)) 20%)`, where the 20 value is equal to the percentage of the way `colorBlue40` is from `colorBlue50` (no tint) to `colorBlue0` (all the way to white).

- This doesn't work so well in cybernetic, where `--white-on-dark` and `--deprecated-accent` are very similar and so almost no contrast is being generated. Redpop's colors, notably, are duplicated by `color-mix(in srgb, rgb(var(--deprecated-accent)), white 20%)`; that is to say, Redpop **always** lightens the color on hover. This sometimes feels like the wrong direction, but it does always generate contrast.